xml/CRSChannelCode.xml

```diff
@@ -9,8 +9,8 @@
         <codelist-item status="active" activation-date="2015-12-07">
             <code>10000</code>
             <name>
-                <narrative>Public Sector Institutions</narrative>
-                <narrative xml:lang="fr">Institutions du Secteur Public</narrative>
+                <narrative>PUBLIC SECTOR INSTITUTIONS</narrative>
+                <narrative xml:lang="fr">INSTITUTIONS DU SECTEUR PUBLIC</narrative>
```

Comment:
capitalisation

Reply:
Thanks, yes, I think it is good to keep the categories in capitals as they are in the OECD DAC spreadsheet.